frontend/pages/create-project.tsx

```diff
@@ -63,99 +64,116 @@ const CreateProject: React.FC = () => {
 
     const formik = useFormik({
         initialValues: {
-            projectName: '',
-            aboutProject: '',
-            selectedFile: null,
-            listOfParticipant: [] as string[],
-            newList: '',
+            name: '',
+            description: '',
+            selectedFile: '',
+            members: [] as string[],
+            newMembers: '',
```

Comment:
Delete this, new members will be add in the members property already

Reply:
è.é